docs/content.zh/docs/ops/state/state_backends.md

```diff
@@ -538,14 +540,11 @@ state.checkpoint-storage: filesystem
 {{< tabs "fsstatebackendmigration" >}}
 {{< tab "Java" >}}
 ```java
-StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
-env.setStateBackend(new HashMapStateBackend());
-env.getCheckpointConfig().setCheckpointStorage("file:///checkpoint-dir");
-
-
-// Advanced FsStateBackend configurations, such as write buffer size
-// can be set by manually instantiating a FileSystemCheckpointStorage object.
```

Comment:
Should we mention the write buffer size here?

The option is: `CheckpointingOptions#FS_WRITE_BUFFER_SIZE`

Reply:
Sure, based on your comment, I have updated the documentation to include the write buffer size option CheckpointingOptions#FS_WRITE_BUFFER_SIZE. Thank you for the suggestion.